src/main/build/debug.h

```diff
@@ -67,8 +67,7 @@ typedef enum {
     DEBUG_FFT,
     DEBUG_FFT_TIME,
     DEBUG_FFT_FREQ,
-    DEBUG_RX_FRSKY_SPI,
-    DEBUG_RX_SFHSS_SPI,
+    DEBUG_RX_SPI,
```

Comment:
This is somewhat problematic, as it will shift the indices of all subsequent debug modes, invalidating the configuration, and requiring version based differentiation in the blackbox log viewer. While I can see merit in this, I am not sure if the benefits outweigh the extra work / complexity that is required to support this outside of the firmware.